### PR TITLE
Add Anchor Links

### PR DESCRIPTION
**Pull request by agude**
Looks like this:

Screenshot from 2021-03-06 15-04-37

### CODE REVIEW

File: assets/_sass/main.scss [hunk 1]
```diff
@@ -121,3 +121,18 @@ blockquote {
     margin-bottom: 0;
   }
 }
+
+// Anchor links
+h1 .anchor-link-img, h2 .anchor-link-img, h3 .anchor-link-img, h4 .anchor-link-img, h5 .anchor-link-img, h6 .anchor-link-img  {
+  visibility: hidden;
+}
+
+h1:hover .anchor-link-img, h2:hover .anchor-link-img, h3:hover .anchor-link-img, h4:hover .anchor-link-img, h5:hover .anchor-link-img, h6:hover .anchor-link-img {
```
Comment (charles-uno) [inline assets/_sass/main.scss:130]:
```suggestion
:hover .anchor-link-img {
```
Does this work?

File: _includes/anchor_headings.html [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+        {% capture node %}<h{{ node }}{% endcapture %}
+      {% endunless %}
+
+      {% capture edited_headings %}{{ edited_headings }}{{ node }}{% endcapture %}
+      {% continue %}
+    {% endif %}
+
+    {% capture _closingTag %}</h{{ headerLevel }}>{% endcapture %}
+    {% assign _workspace = node | split: _closingTag %}
+    {% assign _idWorkspace = _workspace[0] | split: 'id="' %}
```
Comment (charles-uno) [inline _includes/anchor_headings.html:89]: If we're working with ids, does this only work if h1 is unique?

**Review by charles-uno — APPROVED** [review]

File: _includes/anchor_headings.html (hunk 2 above)
Comment (agude) [reply _includes/anchor_headings.html:89]: Jekyll sets the ID of headers to:

```python
id_text = header_text.lower().replace(' ', '-')

# 'I am a header' -> 'i-am-a-header'
```

So they are unique if you don't rename sections.

File: assets/_sass/main.scss (hunk 1 above)
Comment (agude) [reply assets/_sass/main.scss:130]: No.

**Pull request merged**